### PR TITLE
Disable indented code_block

### LINKED ISSUE #82: Enhancement / Bug: Missing CommonMark node type 'code_block'
### What happened?

I added a (test) markdown file that had the following expressions:

```markdown

`inline code!`

~~~text
tilde-fenced-code!
~~~

    # https://spec.commonmark.org/0.30/#indented-code-block
    4-space indented code

```

When I rendered it with the react renderer under nextjs, this document threw the error:

`Error: Undefined node 'code_block'`

### To reproduce

1. Enter the content in a markdown file as described above
1. Render it in your markdoc-based nextjs site
1. See the error!

Will submit a PR with test cases shortly.

### Version

_No response_

### Additional context

I expected this to be supported because the documentation says that the nodes supported are the CommonMark nodes.

That said, it is indeed true that the exact nodes supported table does not include 'code_block' so the documentation may be "technically correct"

Workarounds include:
- Provide my own code_block implementation in my schema
- Avoid validating my markdown document and just swallow the error

### PR DESCRIPTION
**Pull request by mfix-stripe**
Closes #82
Closes #83

### CODE REVIEW

File: src/tokenizer/index.ts [hunk 1]
```diff
@@ -12,7 +12,11 @@ export default class Tokenizer {
     this.parser = new MarkdownIt(config);
     this.parser.use(annotations, 'annotations', {});
     this.parser.use(frontmatter, 'frontmatter', {});
-    this.parser.disable('lheading');
+    this.parser.disable([
+      'lheading',
+      // Disable indented `code_block` support https://spec.commonmark.org/0.30/#indented-code-block
+      'code',
```
Comment (matv-stripe) [inline src/tokenizer/index.ts:18]: does this disable regular fence code blocks too?

image

**Review by matv-stripe — COMMENTED** [review]
question

File: src/tokenizer/index.ts (hunk 1 above)
Comment (mfix-stripe) [reply src/tokenizer/index.ts:18]: Fences are supported by this rule: https://github.com/markdown-it/markdown-it/blob/master/lib/rules_block/fence.js

`code_block` nodes are supported by this rule (with name: `code`): https://github.com/markdown-it/markdown-it/blob/master/lib/rules_block/code.js

Comment (matv-stripe) [reply src/tokenizer/index.ts:18]: huh. That is confusing 😆

**Review by matv-stripe — APPROVED** [review]

**Comment by fbartho** [comment]
Oh! So indented code_block was intentionally disabled? I didn’t catch that in docs!

**Pull request merged**